src/components/NewMessageModal.vue

```diff
@@ -135,7 +135,7 @@ export default {
 				})
 			}
 
-			if (!data.sendAt) {
+			if (!data.sendAt || data.sendAt < Math.floor(now / 1000)) {
```

Comment:
I think this should be a comparison with the UNDO_SEND constant, no? So 30 seconds or whatever?

Reply:
Why do you think so?i don't think so. Here i compare scheduled time (timestamp) from store with timestamp of now. And if it is somehow scheduled, "undo sending" will not appear/